docs/Cypress.md

```diff
@@ -156,6 +156,10 @@ Now you can add `--config=debug` to any `bazel test` command line.
 The runtime will pause before executing the program, allowing you to connect a
 remote debugger.
 
+You can also change the default args that are sent to nodejs. This can be done through a flag. The default is --preserve-symlinks while anything
+can be passed. The flag is --@build_bazel_rules_nodejs//nodejs:default_args="" ex: bazel test --@build_bazel_rules_nodejs//nodejs:default_args="test" main
```

Comment:
Lets add a real flag here to better illustrate what it does. Maybe pass two such as
`"--preserve-symlinks --no-warnings"` to show that you can pass multiple. A drop a link to https://nodejs.org/api/cli.html for reference.